### DISCUSSION TITLE
Returning undefined using Nova::provideToScript into my resource tool

### DISCUSSION THREAD

**Discussion by hugohernandezfcc** [laravel/nova-issues#5367] [Q&A] [answered]
- Laravel Version: ^10.0
- Nova Version: ~4.0
- PHP Version: 8.1
- Database Driver & Version:
- Operating System and Version: ventura
- Browser type and version: google chrome

### Description:
I'm trying access to my variable into my vuejs component using 

## Service provider of my resource tool
`public function boot()
    {
        $this->app->booted(function () {
            $this->routes();
        });

        Nova::serving(function (ServingNova $event) {
            Nova::script('knowledge-content-version', __DIR__.'/../dist/js/tool.js');
            Nova::style('knowledge-content-version', __DIR__.'/../dist/css/tool.css');
        });

        Nova::provideToScript([
            'variablename' => 'test',
        ]);
    }`

## Vuejs
` mounted() {

    Nova.request().get(`/nova-vendor/knowledge-content-version/knowledgeBase/versions/${this.resourceId}`).then(response => {
        console.log(response);
        for (var i = response.data.length - 1; i >= 0; i--) {
          this.versions.push({
            "name": response.data[i].name
          })
        }
        console.log(Nova.config.variablename)
        this.objectSelected.name = response.data[response.data.length - 1].name
      }
    )

  },`

and is returning undefined.

Please help!!

**Comment by crynobone** [comment, marked as answer]
This is an incorrect usage.

```js
console.log(Nova.config.variablename)
```

It should have been the following based on documentation:

```js
console.log(Nova.config('variablename'))
```

**Reply by hugohernandezfcc** [reply]
Thank you so much, finally is running well. @crynobone
